cartoframes/viz/kuviz.py

```diff
@@ -0,0 +1,49 @@
+from carto.kuvizs import KuvizManager
+from carto.exceptions import CartoException
+
+
+class Kuviz(object):
+    PRIVACY_PUBLIC = 'public'
+    PRIVACY_PASSWORD = 'password'
+
+    def __init__(self, context, id, url, name, privacy=PRIVACY_PUBLIC):
+        self.cc = context
+        self.id = id
```

Comment:
:+1:

Reply:
We use `id` everywhere in the Python SDK and it seems we haven't had any problem since now... since this properties are mapping attributes of one our APIs does not look a good idea to me renaming them.

Reply:
OMG

I think you are ready to put a green mark in every PR :)

Reply:
Anyway, I am going to merge them today :stuck_out_tongue: